src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs

```diff
@@ -79,6 +79,58 @@ public class DisplayVersion
             }
         }
 
+        public DisplayVersion(
```

Comment:
This will be deleted, I need to merge https://github.com/NuGet/NuGet.Client/pull/4318 first